packages/styles/components/_c.tooltip.scss

```diff
@@ -90,6 +108,12 @@
         left: 0;
         transform: translate(-99%, -50%);
       }
+
+      &::before {
+        top: 50%;
+        left: -1px;
+        transform: translate(-99%, -50%);
+      }
```

Comment:
The values of the `::before` should be the same as the `::after`.
_(same remark as above)_